tests/test_exceptions.py

```diff
@@ -58,11 +58,26 @@ def test_websockets_should_raise():
 
 
 def test_handled_exc_after_response():
-    # A 406 HttpException is raised *after* the response has already been sent.
-    # The exception middleware should raise a RuntimeError.
-    with pytest.raises(RuntimeError):
```

Comment:
I would change the test much more simply here. A handled exception after the response is rendered isn't a problem now, we should just expect a 200 status code. We don't particularly need to check for side-effects.

Reply:
Wouldn't we want to make sure that the exception handler runs? In case the response is already rendered, we get the same result even if it's not called